content/source/docs/extend/testing/acceptance-tests/sweepers.html.md

```diff
@@ -0,0 +1,121 @@
+---
+layout: "extend-testing"
+page_title: "Extending Terraform - Acceptance Testing"
+sidebar_current: "docs-extend-testing-acceptance-sweepers"
+description: |-
+  Extending Terraform is a section for content dedicated to developing Plugins
+  to extend Terraform's core offering.
+---
+
+# Sweepers
+
+Acceptance tests in Terraform provision and verify real infrastructure using Terraform's testing framework (see [Acceptance Tests](/docs/extend/testing/acceptance-tests/index.html) for more information). Ideally all infrastructure created is then destroyed within the lifecycle of a test, however the reality is that there are several situations that can arise where resources created during a test are “leaked”. Leaked test resources are resources created by Terraform during a test, but Terraform either failed to destroy them at it’s conclusion, or the test falsely reported all resources were destroyed after completing the test. Common causes are intermittent errors or failures in vendor APIs, or developer error in the resource code or test.
```

Comment:
Nitpick:

> using Terraform's testing framework (see [Acceptance Tests](/docs/extend/testing/acceptance-tests/index.html) for more information)

could be

> using [Terraform's testing framework](/docs/extend/testing/acceptance-tests/index.html).

that's shorter and also "better for SEO" :)

Reply:
> failed to destroy them at it’s conclusion

_at it's conclusion_ -> _as part of the test_?

Reply:
Another example which may be worth noting is crash, when the testing framework simply doesn't get chance to perform the cleanup, but I guess that falls under "developer error", so feel free to ignore.